.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,67 @@
+name: "Feature Request"
+description: "Suggest anything you think could improve Treasury."
+labels: [ "enhancement", "unassigned" ]
+
+body:
+
+  - type: "markdown"
+    attributes:
+      value: "## Feature Request"
+
+  - type: "textarea"
+    attributes:
+      label: "User Story"
```

Comment:
I prefer user stories as they usually get people to explain in more detail both what they want and why they want it, including the root cause of their need. I left in the "describe the problem" section in case, but did not make it required. If we don't want a user story we could just make that non-optional or leave it optional and delete this segment.